pkgs/by-name/st/strip-ansi/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  stdenv,
+  buildPackages,
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  nix-update-script,
+  installShellFiles,
+}:
+let
+  canRunStripAnsi = stdenv.hostPlatform.emulatorAvailable buildPackages;
+  stripAnsiCompletions = "${stdenv.hostPlatform.emulator buildPackages} $out/bin/strip-ansi-completions";
+in
+rustPlatform.buildRustPackage {
+  pname = "strip-ansi";
+  version = "0.1.0-unstable-2024-09-01";
+
+  src = fetchFromGitHub {
+    owner = "KSXGitHub";
+    repo = "strip-ansi-cli";
+    rev = "60dbdbc22b41f743c237cb75b11e72cf7884b792";
+    hash = "sha256-FvozEjNWXE1XEIq/06JehES7LVKoWmzIoaB4fD1kUsY=";
+  };
+
+  cargoHash = "sha256-FNtU3FqoFdHIQJNHhIR25zrhjEenj1okfHfVGvprph0=";
```

Comment:
error: hash mismatch in fixed-output derivation '/nix/store/b0w5m3bj3lnfmrryhy81x3kkxpsmy1cl-strip-ansi-0.1.0-unstable-2024-09-01-vendor.tar.gz.drv':
         specified: sha256-FNtU3FqoFdHIQJNHhIR25zrhjEenj1okfHfVGvprph0=
            got:    sha256-oo1GfX2MGRXTSixHgT8kwx8lvQjlTRnedjFlKNP8674=

Reply:
Updated with `useFetchCargoVendor` and regenerated the `cargoHash`.